tests/Doctrine/Tests/Models/ReadonlyProperties/Book.php

```diff
@@ -5,7 +5,6 @@
 namespace Doctrine\Tests\Models\ReadonlyProperties;
 
 use Doctrine\Common\Collections\ArrayCollection;
-use Doctrine\Common\Collections\Collection;
```

Comment:
Why did you remove this? I think that's what makes the tests fail.

Reply:
This line is removed automatically by `phpcbf`

Reply:
Indeed, I experience the same bug when I try your branch. Can you try restoring that line and pushing? Maybe it will work in the CI, as the CI uses PHP 8.1.

Reply:
OK lets see with the CI